# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/dhcp/hotfix.patch.nios2
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- dhcp-4.4.2/bind/bind-9.11.14/lib/isc/stats.c.vanilla	2021-10-09 16:10:43.604479090 +0200
+++ dhcp-4.4.2/bind/bind-9.11.14/lib/isc/stats.c	2021-10-09 16:11:11.538477425 +0200
@@ -297,7 +297,7 @@
 	isc_atomic_store((int32_t *)&stats->counters[counter].lo,
 			 (uint32_t)(value & 0xffffffff));
 # else
-	stats->counters[counter] = val;
+	//stats->counters[counter] = val;
 # endif
 #endif
 }
